### DISCUSSION TITLE
Seit letztem Update funktioniert das HmIP-WRCD Display nicht mehr

### DISCUSSION THREAD

**Discussion by ICEAndreas** [OpenCCU/OpenCCU#2305] [🇩🇪 Fragen&Antworten] [answered]
Seit letztem Update funktioniert das HmIP-WRCD Display nicht mehr korrekt. Tastendrücke werden erkannt aber das Display zeigt nichts mehr an. Nach einem Neustart bleibt es bei  der Anzeige "Homematic" und "Kanal 1+2".

**Comment by MichaelN0815** [comment, marked as answer]
Ist doch schon in Nightly behoben

**Reply by ICEAndreas** [reply]
Wann wird die denn veröffentlicht?

**Reply by jens-maus** [reply]
"It‘s done when it's done" ist da die Devise. Ansonsten gilt wie immer einen Blick in die Milestones im GitHub Projekt zu werfen um eine grobe Richtung zu bekommen (siehe https://github.com/jens-maus/RaspberryMatic/milestones)
